packages/cli/src/lib/config/globalConfig.ts

```diff
@@ -0,0 +1,19 @@
+import {Interfaces} from '@oclif/core';
```

Comment:
Storing Oclif's config into a singleton to prevent extending namespaces

Reply:
Nice, 'tis cleaner too imo